code/lib/core-server/src/utils/get-story-id.ts

```diff
@@ -0,0 +1,44 @@
+import type { Options } from '@storybook/types';
+import dedent from 'ts-dedent';
+import { normalizeStories, normalizeStoryPath } from '@storybook/core-common';
+import path from 'path';
+import { storyNameFromExport, toId } from '@storybook/csf';
+import slash from 'slash';
+import { userOrAutoTitleFromSpecifier } from '@storybook/preview-api';
+
+interface Data {
+  storyFilePath: string;
+  exportedStoryName: string;
+}
```

Comment:
Is this interface different from the others? Either way it should have a descriptive name.